packages/xrpl/test/integration/requests/serverInfo.test.ts

```diff
@@ -138,29 +144,24 @@ describe('server_info (rippled)', function () {
       for (const obj of response.result.info.load?.job_types ?? []) {
         assert.equal(typeof obj.job_type, 'string')
       }
+
       // state_accounting
-      Object.keys(response.result.info.state_accounting).forEach(function (
-        key,
-      ) {
-        assert.equal(
-          typeof response.result.info.state_accounting[key].duration_us,
-          'string',
-        )
-        assert.equal(
-          typeof response.result.info.state_accounting[key].transitions,
-          'string',
-        )
+      const state_accounting: StateAccountingFinal & {
+        [index: string]: unknown
```

Comment:
Why do we need this catch-all type here? I have found that we can iterate over the keys without using this additional key-value pair.